openlibrary/catalog/get_ia.py

```diff
@@ -1,14 +1,17 @@
 from openlibrary.catalog.marc.marc_binary import MarcBinary
 from openlibrary.catalog.marc.marc_xml import MarcXml
 from openlibrary.catalog.marc import parse
+from infogami import config, load_config
 from lxml import etree
 import xml.parsers.expat
 import urllib2, os.path, socket
 from time import sleep
 import traceback
 from openlibrary.core import ia
 
-base = "https://archive.org/download/"
+load_config('conf/openlibrary.yml')
+IA_BASE_URL = config.get('ia_base_url')
+IA_DOWNLOAD_URL = '%s/download/' % IA_BASE_URL
```

Comment:
Some lines seem to assume that `IA_DOWNLOAD_URL` ends w/ a `/` and some don't, e.g.
https://github.com/internetarchive/openlibrary/pull/1058/files/51baf2e92efcf8b5167563ee2e1efc80d5ec6908..54e2cc6e9710e2775135b5e3c29d5e8a61bda499#diff-5af8c98390b7618eb32bb6784bdbc87fR68